indico/web/flask/templating.py

```diff
@@ -53,6 +54,10 @@ def dedent(value):
     return indentation_re.sub('', value)
 
 
+def html2text(value):
+    return EnsureUnicodeExtension.ensure_unicode(clean(value, tags=[], strip=True))
```

Comment:
Doesn't `bleach.clean` just escape the HTML? Shouldn't we remove any tags instead?

Reply:
`strip=Tru`

Reply:
Ah yeah, sorry, didn't scroll enough :)

Reply:
What I am thinking about here is, wether this can/should  be combined with indico.utils.string# text_to_repr() , which adds mainly space removal and truncation on top.

Reply:
I wouldn't. That function has a completely different purpose.

Reply:
```
>>> s = 'foo &amp; bar <a href="test">xxx</a> 1<2 <test>'

>>> from lxml.html import html5parser
>>> html5parser.fromstring(s).xpath('string()')
'foo & bar xxx 1<2 '

>>> import bleach
>>> bleach.clean(s, strip=True, tags=[])
u'foo &amp; bar xxx 1&lt;2 '
```

Maybe using html5parser is better here? We use the description in a plaintext context so having escaped entities etc. is not very nice.

Also, I don't think we need the EnsureUnicodeExtension here; new code already uses unicode. So I would move the new html2text implementation to `indico/util/string.py` and just do the conversion (assuming the input is already unicode). It'd also be a good opportunity to add a unit test for it (`indico/util/string_test.py`, check the other testcases in there for examples)